Add VPN configuration details for different cloud providers (#3806)
astronomer/docs: astro/connect-aws.md

        {label: 'VPC peering', value: 'VPC peering'},
        {label: 'Transit Gateways', value: 'Transit Gateways'},
        {label: 'AWS PrivateLink', value: 'AWS PrivateLink'},

>         {label: 'VPN', value: 'VPN'},

Reviewer: Need to add this option to the list on line 40

</TabItem>

<TabItem value="VPN">

:::info

> This connection option is not available for standard Astro Hosted clusters. It is available for dedicated clusters and Astro Hybrid.

Reviewer: The text here is different from the other info banners

> :::info
> 
> This connection option is not available for standard Astro Hosted clusters. It is available for dedicated clusters and Astro Hybrid.
> 
> :::

Reviewer 2: ```suggestion
:::info
This connection option is only available for dedicated Astro Hosted clusters and Astro Hybrid.
:::
```

</TabItem>

<TabItem value="VPN">

:::info

> This connection option is only available for dedicated Astro Hosted clusters and Astro Hybrid.

Reviewer 2: ```suggestion
This connection option is not available for standard Astro Hosted clusters. It is available for dedicated clusters and Astro Hybrid.
```

:::

> Use this connectivity type for access to on-premises resources or inside onther cloud providers.

Reviewer 2: ```suggestion
Use this connectivity type to access on-premises resources or resources in other cloud providers.
```

#### Prerequisites

> 1. Resources what you need at the beginning:
> 
> - An Astro cluster
> - Configured Gateway device or Application with Public IP address (or 2 addresses for HA setup)
> 
> 2. Contact internal Network team or engineer and ask for the following information:
> 
> - Public IP addresses for the tunnels configuration.
> - IKE pre-shared key, if your team wants to use a particular one.
> - Preferable settings for phase 1 and phase 2 (BGP only) IKE negotionations.
> - ASN for BGP or IP Prefixes for Static configuration.
> - Optional. A size /30 IPv4 CIDR block from the 169.254.0.0/16 range for the inside tunnel IPv4 addresses.

Reviewer 2: ```suggestion

- An Astro Deployment with a dedicated cluster.
- Configured gateway device or application with Public IP address. You need 2 addresses for an HA setup.

Contact your internal network team or engineer and ask for the following information:

- Public IP addresses for the tunnels configuration.
- IKE pre-shared key, if your team wants to use a particular key.
- Preferable settings for phase 1 and phase 2 (BGP only) IKE negotiations.
- ASN for BGP or IP prefixes for static configuration.
- (Optional) A size /30 IPv4 CIDR block from the 169.254.0.0/16 range for the inside tunnel IPv4 addresses.
```

#### Contact Astronomer support for VPN configuration on Astro side

> Submit all collected details via Support portal. CRE team will proceed with the required steps. In case of additional questions or clarifications, CRE team will contact you within Support request or let you know about connectivity tests.

Reviewer 2: ```suggestion
Submit all collected details to [Astronomer support](https://cloud.astronomer.io/open-support-request). The Astronomer CRE team will proceed with the required steps. The CRE team will contact you using your support ticket to ask follow-up questions, request clarification, or let you know about connectivity tests.
```
